layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+<link href="{{ "css/release_binaries.css" | relURL }}" rel="stylesheet">
+<script defer src="{{ "js/release_binaries.js" | relURL }}"></script>
+
+<div class="filter-options">
+    <div class="os-options">
+        <div>
+            <b>{{ T "release_binary_os_option" }}</b>
+            <div class="buttons" id="os-filter">
+                {{- range $AllOSes }}
+                <button class="button" data-os="{{.}}-data">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+    <div class="version-options">
+        <div>
+            <b>{{ T "release_binary_version_option" }}</b>
+            <div class="buttons" id="version-filter">
+                {{- range $AllVersions }}
+                <button class="button" data-version="{{.}}-data">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
+
+    <div class="arch-options">
+        <div>
+            <b>{{ T "release_binary_arch_option" }}</b>
+            <div class="buttons" id="arch-filter">
+                {{- range $AllArch }}
+                <button class="button" data-arch="{{.}}-data">{{.}}</button>
+                {{- end }}
+            </div>
+        </div>
+    </div>
```

Comment:
Extra credit: default to `runtime.PlatformArch` if it matches something in the list.

Not required for an initial PR.

Reply:
Attempted to utilize [runtime.PlatformArch](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/PlatformArch) as a default selection, but it seems to be a part of the browser extension API. I couldn't get it to work as intended because using it in JavaScript within a <script> tag operates within the page's context, where it lacks access to browser level APIs. Perhaps this could indeed be a potential improvement for the future.